otpedittext2/src/main/java/com/broooapps/otpedittext2/OtpEditText.java

```diff
@@ -195,6 +197,12 @@ public String getOtpValue() {
         }
     }
 
+    @NonNull
+    public int getNumCharsMax() {
```

Comment:
Can we have a better name for this method?

Reply:
I thought `getNumChars` was a bit confusing if it was current input num of chars or max value.

And I thought `getMaxLenghth` was not clear enough, length of what?
But it would be ok.

Or we can name `getMaxCharLength` etc,

Reply:
`getMaxCharLength() ` sounds a bit better, imo.
Please make the changes, will approve and publish another version.

Reply:
@swapnil1104 fixed!